static/app/views/monitors/components/monitorCheckIns.tsx

```diff
@@ -71,6 +72,10 @@ function MonitorCheckIns({monitor, monitorEnvs, orgId}: Props) {
   const generateDownloadUrl = (checkin: CheckIn) =>
     `/api/0/organizations/${orgId}/monitors/${monitor.slug}/checkins/${checkin.id}/attachment/`;
 
+  const renderDateTime = (date: string, timezone?: string) => (
```

Comment:
I would probably just in-line this in the tooltip tbh, not a lot of value having it as a function I think?